source/server/admin/admin.cc

```diff
@@ -82,6 +84,27 @@ std::vector<absl::string_view> prepend(const absl::string_view first,
   strings.insert(strings.begin(), first);
   return strings;
 }
+
+Http::HeaderValidatorFactoryPtr createHeaderValidatorFactory(
+    [[maybe_unused]] Server::Configuration::ServerFactoryContext& context) {
+  Http::HeaderValidatorFactoryPtr header_validator_factory;
+#ifdef ENVOY_ENABLE_UHV
+  // Default UHV config matches the admin HTTP validation and normalization config
+  ::envoy::extensions::http::header_validators::envoy_default::v3::HeaderValidatorConfig uhv_config;
+
+  ::envoy::config::core::v3::TypedExtensionConfig config;
+  config.set_name("default_envoy_uhv_for admin");
```

Comment:
nit: this name setting seems completely meanningless?

Reply:
This name is not used anywhere. It is only useful when debugging config rejection if someone builds Envoy with UHV enabled but without UHV extension linked. Hopefully this will provide a clue as to why config is being rejected.